gdal/swig/python/gdal-utils/setup.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Setup script for gdal-utils.
+
+from setuptools import setup, find_packages
+
+__pacakge_name__ = 'gdal-utils'
+__version__ = '3.3.0'
```

Comment:
HOWTO-RELEASE should be updated to ask to update this number in the part where we update version numbers

Reply:
done.
I've added a build for the gdal-utils wheel as well.